risc0/zkvm/src/host/api/mod.rs

```diff
@@ -193,10 +195,14 @@ fn get_server_version<P: AsRef<Path>>(server_path: P) -> Result<Version> {
 
 impl Connector for ParentProcessConnector {
     fn connect(&self) -> Result<ConnectionWrapper> {
+        let passthru_vars: HashMap<_, _> = std::env::vars()
+            .filter(|(key, _)| key == "RUST_LOG" || key == "RUST_BACKTRACE")
+            .collect();
         let addr = self.listener.local_addr()?;
         let child = Command::new(&self.server_path)
             .arg("--port")
             .arg(addr.port().to_string())
+            .envs(passthru_vars)
```

Comment:
Is the intention here to _only_ pass the `passthru_vars`? Is so, I think `env_clear` needs to be called. https://doc.rust-lang.org/std/process/struct.Command.html#method.env_clear

Reply:
I was really just looking for a way to enable logging on the server-side based on the client-side settings. It's not clear that we need to clear the env because otherwise, wouldn't the RUST_LOG have been inherited?

Reply:
Ya, this one is odd to me. Reading the docs, it seems like an env var like `RUST_LOG` should already be inherited from the current process. If that was accurate, this this would do nothing 😕

Reply:
My interpretation is that the environment is already clean, and that we don't need to use `env_clear`.